swag_exporter/setup_info.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "swag_export",
+  "version": "0.0.1",
+  "license": "License :: Other/Proprietary License",
+  "exclude_folders": [],
+  "url": "https://voereir.com",
+  "author": "VoerEir AB",
+  "author_email": "info@voereir.com",
+  "long_description": "Swag Exporter for exporting swagger spec json as html to pdf.",
+  "description": "Swag Exporter for exporting swagger spec json as html to pdf.",
+  "classifiers": ["Programming Language :: Python :: 3.7"],
+  "platforms": ["any"],
+  "python_requires": ">=3.7.4",
+  "entry_points": {
+   "console_scripts": [
+      "VoerEir_Engines = swag.main:main"
```

Comment:
I don't think it should be VoerEir_Engines

Reply:
Ya I do agree. Since it was present before so I added it as it as.

Reply:
Changed its name